### PR TITLE
Update jdtls.lua

### PR DESCRIPTION
**Pull request by nzlov**
multi-module projects first.

### CODE REVIEW

File: lua/lspconfig/server_configurations/jdtls.lua [hunk 1]
```diff
@@ -77,15 +77,15 @@ local function on_language_status(_, result)
 end
 
 local root_files = {
+  -- Multi-module projects
+  { '.git', 'build.gradle', 'build.gradle.kts' },
```
Comment (glepnir) [inline lua/lspconfig/server_configurations/jdtls.lua:81]: why there add `.git`

Comment (nzlov) [reply lua/lspconfig/server_configurations/jdtls.lua:81]: Sorry for such a late reply.
My current project uses `maven` multi-module.
```
.
├── dependencies
├── generator
├── .git
├── .gitignore
├── .idea
├── modules
├── pom.xml
├── .project
├── README.md
├── .settings
├── common
├── starter
└── upms
```

**Comment by Akimcx** [comment]
Hey, just curious, any chance this is getting merged?

**Comment by nzlov** [comment]
Oh...... no. A year has passed.

**Comment by Akimcx** [comment]
Thx you

**Pull request merged**